lib/game/game.dart

```diff
@@ -130,6 +131,21 @@ abstract class Game {
     );
   }
 
+  /// Utility method to load and cache the image for a [SpriteBatch] based on its options
+  Future<SpriteBatch> loadSpriteBatch(
+    String path, {
+    Color defaultColor = const Color(0x00000000),
```

Comment:
Shouldn't this be opaque white?

Reply:
This was the value it already was, fully transparent, because it is used as the background color for the sprites.